pycheribuild/projects/run_fvp.py

```diff
@@ -608,6 +618,49 @@ def add_hostbridge_params(*params):
                 # "-C", "css.trustedBootROMloader.fname=" + str(trusted_fw),
                 "-C", "css.pl011_uart_ap.unbuffered_output=1",
             ]
+
+            if self.fvp_cadiserver:
+                fvp_args += ["-S"]
```

Comment:
How do you connect to it? I guess the client requires Linux so forwarding that for Docker doesn't really make much sense?

Reply:
I'm still unsure how the whole CADI debugging stack works end-to-end but there is no option to specify the port, I only added it to test whether I could get a connection locally, but I guess that for Docker does not make much sense. Not sure if we want to ditch it or disable it?

Reply:
Dropped